migrator/migrations/m_189_to_m_190_vulnerability_requests_add_name/store/previous/store.go

```diff
@@ -0,0 +1,302 @@
+package previous
```

Comment:
I would put these in a `test` folder to make it clear they are only needed for test and not the migration itself.

Reply:
I think it is okay, in that, we do not disallow reading through stores.  It is better to stick to one pattern.